homeassistant/components/fritzbox/sensor.py

```diff
@@ -107,7 +109,7 @@ class FritzSensorEntityDescription(
         device_class=SensorDeviceClass.CURRENT,
         state_class=SensorStateClass.MEASUREMENT,
         suitable=lambda device: device.has_powermeter,  # type: ignore[no-any-return]
-        native_value=lambda device: device.power / device.voltage / 1000
+        native_value=lambda device: round(device.power / device.voltage, 3)
```

Comment:
I know that this sensor violates the rule about calculated sensors, but I don't think we should remove it now.
Rather, I would like tosuggest moving this computation to the pyfritzhome library, if this is acceptable?

Reply:
- for reference hthiery/python-fritzhome/pull/68